backend/test/Test/Parser.hs

```diff
@@ -0,0 +1,89 @@
+module Test.Parser
+  ( spec_parser
+  ) where
+
+import Universum
+
+import qualified Data.ByteString.Lazy as L
+import qualified Data.Set as S
+
+import Codec.Xlsx (CellValue(..), Worksheet, ixSheet)
+import Codec.Xlsx.Parser (toXlsx)
+import Test.Hspec (Spec, describe, it, shouldBe, shouldThrow)
+
+import Edna.ExperimentReader.Error (ExperimentParsingError(..))
+import Edna.ExperimentReader.Parser (processWorkSheet)
+import Edna.ExperimentReader.Types (CellType(..), PointYX(..))
+import Edna.Web.Types (ExperimentalMeasurement(..))
+
+tablesPath :: FilePath
+tablesPath = "./resources/testSheets"
+
+getWorkSheet :: FilePath -> Text -> IO Worksheet
+getWorkSheet filePath sheetName = do
+  file <- L.readFile filePath
+  maybe (throwM WorksheetNotFound) pure $ (toXlsx file) ^? ixSheet sheetName
+
+tableBaseTest :: FilePath -> Text -> [Text] -> [Text] -> Int -> Int -> IO ()
```

Comment:
I think it's semantically more correct to write `Expectation` here instead of `IO ()` (they are synonyms).